pdns/dnsdistdist/xsk.cc

```diff
@@ -0,0 +1,418 @@
+/*
```

Comment:
We need to add xsk.cc and xsk.hh to `dnsdist_SOURCES` in Makefile.am. We will worry later about what happens if XDP/libxdp/... is not available.

Reply:
I have implemented the compile argument `-with-xsk` to decide whether to use `xsk` or not. But it didn't complete the decision to link `libxdp` based on the presence or absence of `-with-xsk`.